src/Sentry.NLog/SentryNLogOptions.cs

```diff
@@ -42,31 +42,42 @@ public int ShutdownTimeoutSeconds
         /// </summary>
         public bool IgnoreEventsWithNoException { get; set; } = false;
 
+        /// <summary>
+        /// Determines whether context-level properties will be sent to sentry as additional data. Defaults to <see langword="true" />.
+        /// </summary>
+        /// <seealso cref="SendEventPropertiesAsTags" />
+        public bool SendContextPropertiesAsData { get; set; } = true;
+
+        /// <summary>
+        /// Determines whether event-level properties will be sent to sentry as additional data. Defaults to <see langword="true" />.
+        /// </summary>
+        /// <seealso cref="SendEventPropertiesAsTags" />
+        public bool SendEventPropertiesAsData { get; set; } = true;
```

Comment:
Remove this property or instead map it to `TargetWithContext.IncludeEventProperties` in `InitializeTarget`.